src/what-about-safety.md

```diff
@@ -0,0 +1,58 @@
+# So it's fast, what about safety?
+
+---
+
+## Rust is *memory safe*
+
+* It applies bounds checks to array and slice accesses
+* You can 'accidentally' go off the end of a buffer
+* Massive reduction in bugs
+* Where possible (e.g. the indicies are constant) those checks are optimised out
+
+---
+
+## Rust is *thread-safe*
+
+* You cannot create race-hazards!
+* Every value has one owner
+* You can create either:
+  * One exclusive reference
```

Comment:
I was in support of renaming the mutable/immutable variables for the stated reasons, but here without much context I found the renaming very confusing, because `exclusive` and `shared` says nothing about mutability, and it's easy to read the exact opposite into them: shared is mutable by many, exclusive is not mutable. And I think the mutability part is still an important distinction. This is probably worth some more internal discussion.

Reply:
Attempted to clarify this.